index.d.ts

```diff
@@ -39,6 +39,7 @@ export declare namespace BenchNode {
 
 	type BenchmarkFunction = (timer?: {
 		start: () => void;
+		end: (iterations: number) => void;
```

Comment:
That's optional as timer.end() implies iterations=1.

Reply:
ah good catch, sorry about that - updated this 👍